### DISCUSSION TITLE
Binary visualization - "0" should also mean "No"

### DISCUSSION THREAD

**Discussion by brockfanning** [open-sdg/open-sdg#1449] [Ideas]
**Is your feature request related to a problem? Please describe.**

Our binary visualization is coded to require that "No" is in the data as "-1". However in the SDGs, the actual instructions for countries are to use "0", and not "-1".

**Describe the solution that you need**

Let's support both "0" and "-1" for "No".

**Why do you need this feature?**

In order to visualize SDG Yes/No data exactly as it is reported.

**Describe alternatives you've considered**

One alternative is to rely on countries to add Python code that changes all their 0 values to -1, for certain yes/no indicators. The problem with this is that it requires programming expertise and adds to the maintenance burden.

**Comment by brockfanning** [comment]
@brockfanning Link to the evidence that this is actually the standard.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning is this difficult to implement or is it case of adding or statement here: https://github.com/open-sdg/open-sdg/blob/1.5.0-dev/_includes/components/charts/binary.html#L15

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning what was the reasoning behind using -1? Was it just to force it for the chart?

Do you think worth just changing to 0 (and removing -1 completely?) for 2.0.0? Just thinking binary is normally 0 and 1 in general, not just for SDG reporting
